src/__tests__/reactUtils-test.js

```diff
@@ -0,0 +1,232 @@
+        { waitBefore: MS / 2, expectedOutput: false, thenUpdateInputTo: true },
+        { waitBefore: MS / 2, expectedOutput: false, thenUpdateInputTo: false },
+        { waitBefore: MS / 2, expectedOutput: false, thenUpdateInputTo: true },
+        { waitBefore: MS / 2, expectedOutput: false, thenUpdateInputTo: false },
+        { waitBefore: MS * 2, expectedOutput: false },
+      ],
+    },
+    {
+      initialValue: false,
+      sequence: [
+        { waitBefore: MS / 5, expectedOutput: false, thenUpdateInputTo: false },
+        { waitBefore: MS / 5, expectedOutput: false, thenUpdateInputTo: false },
+        { waitBefore: MS / 5, expectedOutput: false, thenUpdateInputTo: false },
+        { waitBefore: MS / 5, expectedOutput: false, thenUpdateInputTo: false },
+        { waitBefore: MS / 5, expectedOutput: false, thenUpdateInputTo: false },
+        { waitBefore: MS / 5, expectedOutput: false, thenUpdateInputTo: false },
+      ],
+    },
+    {
+      initialValue: true,
+      sequence: [{ waitBefore: MS / 2, expectedOutput: false }],
+    },
+    {
+      initialValue: true,
+      sequence: [{ waitBefore: MS * 2, expectedOutput: true }],
+    },
+    {
+      initialValue: true,
+      sequence: [
+        { waitBefore: MS / 2, expectedOutput: false, thenUpdateInputTo: false },
+        { waitBefore: MS, expectedOutput: false },
+      ],
+    },
+    {
+      initialValue: true,
+      sequence: [
+        { waitBefore: MS * 2, expectedOutput: true, thenUpdateInputTo: false },
+        { waitBefore: MS / 2, expectedOutput: false },
+        { waitBefore: MS, expectedOutput: false },
+      ],
+    },
+    {
+      initialValue: true,
+      sequence: [
+        { waitBefore: MS / 2, expectedOutput: false, thenUpdateInputTo: false },
+        { waitBefore: MS / 2, expectedOutput: false, thenUpdateInputTo: true },
+        { waitBefore: MS / 2, expectedOutput: false, thenUpdateInputTo: false },
+        { waitBefore: MS / 2, expectedOutput: false, thenUpdateInputTo: true },
+        { waitBefore: MS / 2, expectedOutput: false, thenUpdateInputTo: false },
+        { waitBefore: MS * 2, expectedOutput: false },
+      ],
+    },
+    {
+      initialValue: true,
+      sequence: [
+        { waitBefore: MS / 5, expectedOutput: false, thenUpdateInputTo: true },
+        { waitBefore: MS / 5, expectedOutput: false, thenUpdateInputTo: true },
+        { waitBefore: MS / 5, expectedOutput: false, thenUpdateInputTo: true },
+        { waitBefore: MS / 5, expectedOutput: false, thenUpdateInputTo: true },
+        { waitBefore: MS / 5 - 1, expectedOutput: false, thenUpdateInputTo: true },
+        { waitBefore: MS / 5 + 1, expectedOutput: true },
+      ],
+    },
+  ];
+
+  for (let i = 0; i < sequencesToTest.length; i++) {
+    const currentSequence = sequencesToTest[i];
+    test(JSON.stringify(currentSequence, null, 2), async () => {
+      await testSequence(sequencesToTest[i]);
```

Comment:
nit:
```suggestion
    const currentSequence = sequencesToTest[i];
    test(JSON.stringify(currentSequence, null, 2), async () => {
      await testSequence(currentSequence);
```
Or is there something subtle here I'm missing?

Reply:
Nope, you're right! Thanks!